src/components/Game.js

```diff
@@ -86,25 +84,22 @@ class Game extends Component {
       filteredCountries = this.state.allCountries
     }
 
+    let maxCountries = (filteredCountries.length < 20) ? filteredCountries.length : 20
```

Comment:
I think you're right that this is gonna improve the user experience a LOT. It's no fun always hunting for fruitless coordinates!